src/core/settings.py

```diff
@@ -241,8 +243,9 @@ def _load_api_raml(path):
 
 INGESTION_LOG_NAME = 'ingestion-%s.log' % PROJECT_NAME
 
-LOG_FILE = join('/var/log/', LOG_NAME) # ll: /var/log/lax.log
-INGESTION_LOG_FILE = join('/var/log/', INGESTION_LOG_NAME) # ll: /var/log/lax.log
+LOG_DIR = '/var/log/' if ENV != DEV else './'
```

Comment:
there was some problem here with the smoke tests being broken on the Vagrant VM, let me retrieve what was that